vocabularies/Common.xml

```diff
@@ -100,7 +101,7 @@
         <Annotation Term="Core.Description" String="An identifier to distinguish multiple texts in different languages for the same entity" />
       </Term>
 
-      <Term Name="TextFormat" Type="Common.TextFormatType" AppliesTo="Property Parameter ReturnType">
+      <Term Name="TextFormat" Type="Common.TextFormatType" Nullable="false" AppliesTo="Property Parameter ReturnType">
```

Comment:
```suggestion
      <Term Name="TextFormat" Type="Common.TextFormatType" Nullable="true" AppliesTo="Property Parameter ReturnType">
```
This should behave like `Text`, i.e., be nullable.

Reply:
TextFormat is an enumeration with two values: `plain` or `html`. What should `null` mean?

Reply:
Then the description is wrong:
> Human-readable text that may contain formatting information.

Should be:
> Format in which human-readable text may contain formatting information.

Reply:
Changed description